swift_scality_backend/http_utils.py

```diff
@@ -105,12 +107,24 @@ def __init__(self, sock, debuglevel=0, strict=0, method=None,
                     'buffering': buffering,
                 })
 
-            httplib.HTTPResponse.__init__(self, **init_args)
-
+            # Prevent makefile calling by stdlib
+            makefile = socket._socketobject.makefile
+            socket._socketobject.makefile = lambda *_, **_2: None
+            try:
+                httplib.HTTPResponse.__init__(self, **init_args)
+            finally:
+                socket._socketobject.makefile = makefile
+            # Handle eventet subtlety
```

Comment:
Consider providing a reference to the original source code, and change the code into an actual source (same variable names etc).